### ISSUE TITLE
Xdebug on Linux dies Not work with Version 3 because the variable xdebug.client_host has changed. This needs to be used if the Xdebug Remote Host env isset

### ISSUE THREAD

**Issue by martinweinmayr** [dockware/dockware#148]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Blackstareye** [comment]
I can elaborate on that.

TLDR;
```yaml
environment:
    - XDEBUG_REMOTE_HOST=172.17.0.1
```
must be set in docker-compose.yaml if the os is **linux-based**.

so my environment-section looks like that:

TLDR; config that worked
![Bildschirmfoto vom 2023-05-03 15-29-39](https://user-images.githubusercontent.com/33430997/235935874-1aeb1fd2-17d4-4e23-8e09-4fc85408a792.png)




Here a little  ana-data (used with 'dockware/dev:6.4.20.0'; PHP_VERSION=8.1)

Without this environment constant, the output of  `cat /etc/php/8.1/fpm/conf.d/20-xdebug.ini` looks like that:
![Bildschirmfoto vom 2023-05-03 15-27-51](https://user-images.githubusercontent.com/33430997/235934861-d6041041-841d-4ee1-96b4-47805ccd7828.png)

<hr>
line `xdebug.client_host` is the interesting one here
<hr>

With 'XDEBUG_REMOTE_HOST=172.17.0.1' set, the output of `cat /etc/php/8.1/fpm/conf.d/20-xdebug.ini` looks like that:

![Bildschirmfoto vom 2023-05-03 15-27-36](https://user-images.githubusercontent.com/33430997/235935425-939e0a8e-cffd-4a89-b3f2-290ca2597c17.png)

So I think the documentation should mention that for linux users, or this is a bug regarding xdebug usage in dockware.

<hr>

**Comment by boxblinkracer** [comment]
I think we have it
but maybe we should make it more prominent?

https://docs.dockware.io/development/debugging

<img width="915" alt="Screenshot 2023-05-03 at 16 13 26" src="https://user-images.githubusercontent.com/5579326/235942015-325799be-f086-4a2c-84e8-a7350c3085da.png">
<img width="853" alt="Screenshot 2023-05-03 at 16 14 20" src="https://user-images.githubusercontent.com/5579326/235942258-fa8a1a7c-86e6-4f18-92ef-2e83391b9e9d.png">

**Comment by Blackstareye** [comment]
Oh yeah I didn't see that. Maybe we should put it into the info box as well ? 
And in addition ... maybe we should rewrite the box so that it is more clear, what environment variables should be set? 

like:
'Both MAC and Windows have the Docker variable "host.docker.internal" as default value, which should work great as Loopback IP to automatically recognize the host IP.

For Linux however this does not work. Please use the following environment variables instead:
   * XDEBUG_REMOTE_HOST=172.17.0.1'
   * XDEBUG_CONFIG=client_host=172.17.0.1


Maybe something like that? or is this too 'techy' ?

It would pinpoint the actual solution to the problem, though.

What are your thoughts about that?

**Comment by Blackstareye** [comment]
or just similar like in the `WSL`  infobox ?
![grafik](https://user-images.githubusercontent.com/33430997/235945525-f7867051-8b8d-46e0-99d7-dcc834d20dfd.png)

**Comment by boxblinkracer** [comment]
yeah absolutely
if you say the box would be fine, then i would definitely go with copy paste samples :)

ill change it a bit later today

**Comment by Blackstareye** [comment]
Sounds good to me, and many thanks for the quick reply :D

**Comment by Blackstareye** [comment]
Saw that you fixed it already, so I think we can close this issue :) ? 
Really thank you 

PS: (no outcalling -  but nice yt talk :)  )

**Comment by boxblinkracer** [comment]
hey there,
yep, fixed it before the stream on yt, so we can close it yes

haha and thank you, means a lot, really :)
